playtron-os-files/playtron-os-files.spec

```diff
@@ -74,6 +74,9 @@ cp playtron-os-files-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron
 %post
 %systemd_post clatd-ipv6-check.service create-swap.service sddm.service NetworkManager-wait-online.service resize-root-file-system.service inputplumber.service firewalld.service
 %systemd_user_post playserve.service gamescope-dbus.service
+systemd-hwdb update
```

Comment:
The `systemd-hwdb update` works in a container build but the two `udevadm` commands don't. We can either drop those or add some kind of check.

```
systemd-hwdb update
if [ ! -f /usr/bin/rpm-ostree ]; then
    udevadm control --reload-rules
    udevadm trigger
fi
```

Reply:
I removed the udevadm triggers :+1: